components/ntp_background_images/browser/features.h

```diff
@@ -18,13 +18,13 @@ BASE_DECLARE_FEATURE(kBraveNTPSuperReferralWallpaper);
 
 BASE_DECLARE_FEATURE(kBraveNTPBrandedWallpaper);
 
-// Show initial branded wallpaper after nth new tab page for fresh installs.
+// Show initial branded wallpaper after nth new tab page for fresh opens.
 constexpr base::FeatureParam<int> kInitialCountToBrandedWallpaper{
-    &kBraveNTPBrandedWallpaper, "initial_count_to_branded_wallpaper", 1};
+    &kBraveNTPBrandedWallpaper, "initial_count_to_branded_wallpaper", 0};
 
 // Show branded wallpaper every nth new tab page.
 constexpr base::FeatureParam<int> kCountToBrandedWallpaper{
-    &kBraveNTPBrandedWallpaper, "count_to_branded_wallpaper", 3};
+    &kBraveNTPBrandedWallpaper, "count_to_branded_wallpaper", 2};
```

Comment:
I would recommend leaving the default value and testing this initial count via this Griffin feature

Reply:
If we go to 2 I'm not sure I see us have the desire to go back to 3, since numbers would be worse and UX impact doesn't seem huge? Also it's less work and moving pieces in different repos syncing to when this PR hits various channels if we can combine with this PR, since we have to uplift anyway.